pages/firststeps.md

```diff
@@ -35,7 +35,7 @@ There is no official deadline to complete these steps, but most candidates who a
 
     ![Follow Screenshot](uploads/images/Follow.png)
 
-3. We use Crowdin to translate our user interface in many different languages. If you know another language, go to [this Crowdin link](https://crowdin.com/project/open-learning-exchange/invite) to join Crowdin by creating an account. You may start contributing on Crowdin after finishing [Step 4 - BeLL-Apps Tutorial](#!./pages/firststeps.md#Step_4_-_BeLL-Apps_Tutorial) so that you have a better understanding of what BeLL Apps do. You may also reference the [Planet User Manual](http://open-learning-exchange.github.io/#!pages/planetusermanual.md) for detailed explanation of various terms. It would be very helpful for you to translate some of the words and familiarize yourself with the software. It would be great to get at least 10% on a language that you speak. If you want to spend additional time, feel free to provide translations to 100%.
+3. We use Crowdin to translate our user interface in many different languages. If you know another language, go to [this Crowdin link](https://crowdin.com/project/open-learning-exchange/invite) to join Crowdin by creating an account. You may start contributing on Crowdin after finishing [Step 4 - BeLL-Apps Tutorial](#!./pages/firststeps.md#Step_4_-_BeLL-Apps_Tutorial) so that you have a better understanding of what BeLL Apps do. You may also reference the [Planet User Manual](#!./pages/planetusermanual.md) for detailed explanation of various terms. It would be very helpful for you to translate some of the words and familiarize yourself with the software. It would be great to get at least 10% on a language that you speak. If you want to spend additional time, feel free to provide translations to 100%.
```

Comment:
Actually it will work without `#!./pages/`, it would be great if you could fix both links:
`[Step 4 - BeLL-Apps Tutorial](#Step_4_-_BeLL-Apps_Tutorial)`
`[Planet User Manual](planetusermanual.md)`

I added `[Step 4 - BeLL-Apps Tutorial](#!./pages/firststeps.md#Step_4_-_BeLL-Apps_Tutorial)` a few month back without realizing [this section talking about relative urls in readme.md](https://github.com/open-learning-exchange/open-learning-exchange.github.io#relative-urls). I added relative link the wrong way, sorry about the confusion.

It seems like `[installation page](installation.md)` would give `http://open-learning-exchange.github.io/#!./pages/installation.md` in the html, that where I got the `#!./pages/`part.

[MDwiki official documentation about relative link](http://dynalon.github.io/mdwiki/#!quickstart.md#Creating_links)

Reply:
So, since that pull request has been already merged, should i create a new one or what would you advise to me?

Reply:
@MadagascarOLE Yeah sure, create an issue then start a new PR